x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -378,6 +378,59 @@ export class TimeSeriesExplorer extends React.Component {
       );
   };
 
+  /**
+   * Loads available entity values.
+   * @param {Array} entities - Entity controls configuration
+   * @param {Object} searchTerm - Search term for partition, e.g. { partition_field: 'partition' }
+   */
+  loadEntityValues = async (entities, searchTerm = {}) => {
```

Comment:
This code is not a part of `timeseriesexplorer` anymore. You probably got some issues after merging with the latest master.

Reply:
Thanks for pointing it out. I missed this piece during the merge. Removed it here https://github.com/elastic/kibana/pull/81490/commits/3fb8c08929d8209b880f27b05db556e6a21c195b